### ISSUE TITLE
Possible air-raid siren simulation?

### ISSUE THREAD

**Issue by ElijahHamilton** [ange-yaghi/engine-sim#394]
Hi, would it be possible to use the simulation to simulate and air-raid siren?
Thanks

**Comment by BoogieHasFun** [comment]
i don't think it could simulate it perfectly but you could try to make an engine sound like one.
